docs/_data/nav_docs.yml

```diff
@@ -10,6 +10,10 @@
       title: Rendering Elements
     - id: components-and-props
       title: Components and Props
+    - id: lists-conditional-rendering
```

Comment:
I think state should come before lists, but events should come before forms, so maybe interleave them:

state
lists
events
forms

?